components/CreateEventForm/CreateEventForm.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { StyleSheet, View, Button, Text, TextInput} from 'react-native';
+
+export default function CreateEventForm() {
+  return (
+    <View style={styles.container}>
+      <View style={styles.closeButton}><Button title='X'/></View>
+      <Text style={styles.text}>CREATE A NEW EVENT FORM!</Text>
+      <TextInput style={styles.input} placeholder='Event Name...'/>
+      <TextInput style={styles.input} placeholder='Location...'/>
+      <TextInput style={styles.input} placeholder='Start Time...'/>
+      <TextInput style={styles.input} placeholder='End Time...'/>
```

Comment:
We'll want to make sure to limit these fields to numbers only

Reply:
That's smart! Yeah, I was wondering if it would be possible to use a number input that offered arrows for the user to press to increment.